### ISSUE TITLE
Unread Notification count not showing in app icon

### ISSUE THREAD

**Issue by jerryn70** [michelelacorte/FlickLauncher#39]
Unread Notification count is not showing in app icon.  
Checked message, gmail, alto mail , whatsapp , phone but not showing unread count. 

My Phone : moto x play 
Marshmallow 6.0.1

**Comment by neolovich** [comment]
Same problem. Android 5.1.

**Comment by michelelacorte** [comment]
Have you enabled Notification Access from system settings?

**Comment by neolovich** [comment]
Yes, that solved the problem. A bit. The dot seems to work without problems but the unread notification count shows top much or less unread messages. For example the E-Mail App, K-9, has two unread messages although there are zero new Mails. Or for Telegram it shows unread messages are zero but there is one unread messages. And showing that there are zero unread messages cant be the sense of the function :D

**Comment by jerryn70** [comment]
It will be great if notification access is automatically turn on while installing. Is that possible ? I dont think so 🤔 . 
Maybe after installation when the user open launcher settings we can display a splash screen showing to active notification access .

And about this bug. After turning on the notification access , it shows wrong unread count.

**Comment by michelelacorte** [comment]
mmm automatically is not possible, but I can evaluate possibility to add a popup message, otherwise I will fix 0 count soon!

**Comment by mariusmilosan** [comment]
Hi, I have a similar problem: the notification seems not to work for the phone app only. The rest is ok. Please see screenshot.
Using Samsung S7 edge / android 7.0 stock ROM
Thanks!
![screenshot_20170523-042049](https://cloud.githubusercontent.com/assets/28881068/26334709/ccaa65e0-3f6f-11e7-9ea2-3a76370523b0.png)

**Comment by michelelacorte** [comment]
You have enabled Android O notification dots.. and it show!! see dots on icons

**Comment by mariusmilosan** [comment]
On all the other apps it's ok, but on the phone app it does not appear. In
the screen shot I have one SMS unread, one WhatsApp (all ok) and also one
missed call which does not appear as a dot on the phone icon...

**Comment by michelelacorte** [comment]
It appears on the message icon..the package name is the same right?

**Comment by mariusmilosan** [comment]
So the notification dots are working on the other apps (SMS WhatsApp and so
on). The only problem is on the phone app. Sorry but I don't know about
what package name you are talking about. If you ask about the icon pack,
I'm using the GS 8 icons.

Great work btw on the launcher. I love it! 😀

On 23 May 2017 11:46, "Michele Lacorte" <notifications@github.com> wrote:

It appears on the message icon..the package name is the same right?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/michelelacorte/FlickLauncher/issues/39#issuecomment-303332711>,
or mute the thread

**Comment by neolovich** [comment]
Until now the unread notification doesnt work properly. The number of unread messages isnt correct and it shows also unread notifications for Apps like VLC where it doesnt make sense.

**Comment by michelelacorte** [comment]
Because previous method catch all notification (also error etc..) I've improved that!

**Comment by faisalvv** [comment]
some mobile is not support the badge count. actually  badge count is not support the android os. the customized from mobile company. please see link :https://github.com/leolin310148/ShortcutBadger

**Comment by vannt1991** [comment]
> some mobile is not support the badge count. actually badge count is not support the android os. the customized from mobile company. please see link :https://github.com/leolin310148/ShortcutBadger

@michelelacorte pls support badger icon for https://github.com/leolin310148/ShortcutBadger, thanks!

**Comment by GlennTorres** [comment]
Hi! Just downloaded your launcher last night. I have to say that it is an amazing app! However, your unread notication count doesn't work on my end. The settings are all okay -- notification access is granted and the notification dot as well. What could be the problem? Please help. I really love this launcher. Thank you.
